exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/ServiceArtifactIdTest.java

```diff
@@ -26,9 +26,9 @@
 
   @ParameterizedTest
   @ValueSource(strings = {
-      "0:land-registry",
+      "0:land-registry:1.1",
       "1:com.acme:foo-service:1.1.1-beta1",
-      "100500:::",
+      "100500:not-blank:version",
   })
   void parseFromRoundtrip(String serviceId) {
```

Comment:
Must better be the other way around:
`ServiceArtifactId -- toString --> String -- parseFrom --> ServiceArtifactId* -> assertEquals`